app-backend/batch/src/main/scala/cogMetadata/RasterSourceMetadataBackfill.scala

```diff
@@ -91,17 +99,26 @@ object RasterSourceMetadataBackfill extends Job with RollbarNotifier {
 
       implicit val contextShift: ContextShift[IO] = rasterIO
 
-      val response = scenesToUpdate.parTraverse {
+      val metadataIOs = scenesToUpdate.parTraverse {
         case (id, uri) =>
           for {
-            backsplashInfo <- getBacksplashInfo(uri)
-            rmdResult <- insertRasterSourceMetadata(id, uri, backsplashInfo)
-            geotiffResult <- insertGeotiffInfo(id, backsplashInfo)
+            backsplashInfo <- getBacksplashInfo(id, uri)
+            rmdResult <- backsplashInfo.traverse(
+              insertRasterSourceMetadata(id, uri, _))
+            geotiffResult <- backsplashInfo.traverse(insertGeotiffInfo(id, _))
           } yield (rmdResult, geotiffResult)
       }
 
-      response
-        .map(rsm => println(s"Inserted metadata for ${rsm.length} resources"))
+      metadataIOs
+        .map(metadataInserts => {
+          val (lefts, rights) =
+            metadataInserts.partition(t => t._1.isLeft || t._2.isLeft)
```

Comment:
I think tuple access in this way can quickly become cryptic and hard to decipher. This is kind of a nitpicky style suggestion - others on the team might have a preference, but I would prefer the following:

```suggestion
            metadataInserts partition {
              case (Right(_), Right(_)) => false
              case _                    => true
            }
```